src/lib/airtable/tables/requests.test.js

```diff
@@ -0,0 +1,483 @@
+const mockDestroyFn = jest.fn();
```

Comment:
These looks so good! I think this approach makes sense and will definitely help prevent any avoidable breakages in the future.

Just one thing, and this doesn't have to be fixed in this PR but I just want to bring to your attention:

I ran test coverage (`npx jest src/lib --coverage`) and it looks like most of the code is tested, but we are missing tests for a number of error cases

For reference, the `--coverage` output:

```
--------------------------|---------|----------|---------|---------|-----------------------------------
File                      | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s
--------------------------|---------|----------|---------|---------|-----------------------------------
All files                 |   74.01 |    68.75 |      80 |   73.86 |
 donorPayments.js         |   86.67 |      100 |     100 |   86.67 | 34-35
 donorPaymentsSchema.js   |     100 |      100 |     100 |     100 |
 paymentRequests.js       |   21.28 |        0 |   16.67 |   21.28 | 11-35,40-55,60-71,76-79,84-98,106
 paymentRequestsSchema.js |     100 |      100 |     100 |     100 |
 requests.js              |   92.86 |    96.88 |     100 |   92.78 | 49-50,89,115,149,153,212
 requestsSchema.js        |     100 |      100 |     100 |     100 |
--------------------------|---------|----------|---------|---------|-----------------------------------
```

^ most of the uncovered lines for requests.js are error cases.

Reply:
This is a good point, the only reason I didn't shoot for 100% coverage is that this PR was taking long enough and I didn't want it to be too monumental.